UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -148,65 +192,91 @@ void Awake()
 		{
 			foreach (BodyPartSprites b in GetComponentsInParent<BodyPartSprites>())
```

Comment:
Look like this got left and somehow

Reply:
Hmm, strange.  Not my code, and it wasn't firing anyway because I never saw it while testing, removed.